### DISCUSSION TITLE
Vortice.Windows.Samples not found ?

### DISCUSSION THREAD

**Discussion by MarcioAB** [amerkoleci/Vortice.Windows#386] [General]
https://github.com/amerkoleci/Vortice.Windows.Samples ... "page not found".

**Comment by amerkoleci** [comment]
Hi,
It's sponsor only repo now, I'll add it you the repo tho.

Cheers!

**Comment by MarcioAB** [comment]
Thank you. 
🙋‍♂️ +1 small sponsor of your great work.

**Reply by amerkoleci** [reply]
Thanks! Appreciated

**Comment by sdcb** [comment]
Hi @amerkoleci , I'm also a sponsor, please also add me!

**Reply by amerkoleci** [reply]
You're actually not a sponsor, but I added you the same.

Cheers!

**Reply by zaafar** [reply]
lol :D
